appveyor.yml

```diff
@@ -58,7 +58,7 @@ install:
   - cmd: conda update -q -y --all
   - cmd: conda install -q -y numba
   - cmd: conda install -q -y -c conda-forge numpy scipy seaborn networkx pandas pyyaml pytables six sympy pytest
-  - cmd: conda install -q -y sqlalchemy psycopg2 pyarrow traitlets
```

Comment:
Are tests that require pyarrow (for parquet format support I guess?) disabled/skipped on Win builds/tests?

Reply:
An early windows build failure from this branch was complaining about pyarrow. I can add it back to see what it says this time.